src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -18,7 +22,37 @@ public void showNumberFormatError() {
         System.out.println("The input is not a number! Please try again.");
     }
 
+    public void showCommandNotFoundError() {
+        System.out.println("Not a valid command!");
+    }
+
     public void showFarewellMessage() {
         System.out.println("Goodbye! Hope to see you again! :)");
     }
+
+    /**
+     * Creates a scanner in the Ui class
+     */
+
+    public void createScanner() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Gets the next line of user input
+     * 
+     * @return the next of user input
+     */
+
+    public String getNextLine() {
+        return scanner.nextLine();
+    }
+
+    /**
+     * Closes the scanner
+     */
+
+    public void closeScanner() {
+        this.scanner.close();
+    }
```

Comment:
I wonder if there is a way to automatically close a scanner once a function is done using it, but probably not. For now, we will need to manually close it ourselves.